src/index.tsx

```diff
@@ -121,6 +137,13 @@ function Violation({
     };
   }, [open, target]);
 
+  React.useEffect(() => {
+    overlayRef?.current?.setAttribute(
+      'aria-label',
+      `${open ? 'Close' : 'Open'} violation overlay`
```

Comment:
I dunno if this is necessary or not. Is it okay to just have `Toggle violation overlay` instead?

Reply:
I think more explicit is generally better. I'd keep what you have,